solutions/devsprint-riccardo-1/OnboardingChallengeKit/Sources/OnboardingChallengeKit/Screens/Detail/DetailView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DetailView.swift
+//  
+//
+//  Created by Mateus Nazário Coelho on 23/12/21.
+//
+
+import UIKit
+
+final class DetailView: UIView {
+    // MARK: Outlets
+    private lazy var licenseView: LicenseView = {
+        let licenseView = LicenseView()
+        licenseView.translatesAutoresizingMaskIntoConstraints = false
+        return licenseView
+    }()
+    
+    // MARK: Initialization
+    init() {
+        super.init(frame: .zero)
+        setupViews()
+        setupLicenseView()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Methods
+    private func setupViews() {
+        configureSubViews()
+        configureSubviewsConstraints()
+        configureAdditionalBehaviors()
+    }
+    
+    private func configureSubViews() {
+        addSubview(licenseView)
+    }
+    
+    private func configureSubviewsConstraints() {
+        NSLayoutConstraint.activate([
+            self.licenseView.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+            self.licenseView.trailingAnchor.constraint(equalTo: self.trailingAnchor),
+            self.licenseView.topAnchor.constraint(equalTo: self.topAnchor)
```

Comment:
Use `safeAreaLayoutGuide` for the topmost item. This will change as soon as we get the RepositoryInfo and the OwnerCell, but it's better to put it here so the colleague who will do the task would know about them.